src/Subscriptions/SubscriptionPhase.php

```diff
@@ -675,24 +680,23 @@ public static function align( self $phase, \DateTimeInterface $align_date ) {
 
 		$alignment_phase = new self( $phase->get_subscription(), $start_date, $alignment_interval, $phase->get_amount() );
 
-		$alignment_phase->set_total_periods( 1 );
-		$alignment_phase->set_alignment_rate( $alignment_difference->days / $regular_difference->days );
-
-		// Remove one period from regular phase.
-		$total_periods = $phase->get_total_periods();
+		$alignment_end_date = $start_date->add( $alignment_interval );
 
-		if ( null !== $total_periods ) {
-			$phase->set_total_periods( $total_periods - 1 );
-		}
-
-		$alignment_end_date = $alignment_phase->get_end_date();
+		$alignment_phase->set_end_date( $alignment_end_date );
+		$alignment_phase->set_alignment_rate( $alignment_difference->days / $regular_difference->days );
 
-		if ( null === $alignment_end_date ) {
+		if ( null === $alignment_phase->get_end_date() ) {
```

Comment:
This is no longer needed? A few lines earlier i see:

```php
$alignment_phase->set_end_date( $alignment_end_date );
```

Previously this worked indirectly via `$alignment_phase->set_total_periods( 1 );`, this double check was nice, but that has now been eliminated.

Reply:
I noticed the same but forgot to remove it 🙈 Removed in https://github.com/pronamic/wp-pay-core/pull/183/commits/f9d41f14feb717496546b59b6d11f92a2835c079.